index.bs

```diff
@@ -79,8 +79,9 @@ other operating systems. Additionally underlying file systems might have further
 on what names are or aren't allowed, so a string merely being a [=valid file name=] is not
 a guarantee that creating a file or directory with that name will succeed.
 
-A <dfn lt="file|file entry">file entry</dfn> additionally consists of <dfn for="file entry">binary
-data</dfn> and a <dfn for="file entry">modification timestamp</dfn>.
+A <dfn lt="file|file entry">file entry</dfn> additionally consists of
+<dfn for="file entry">binary data</dfn> (a [=byte sequence=]) and a
+<dfn for="file entry">modification timestamp</dfn> (a number representing the number of milliseconds since the <a spec=FileAPI>Unix Epoch</a>).
```

Comment:
(fine to address in the future)

Would it be worth a non-normative note that the format (number of milliseconds since epoch) interops with JavaScript's Date type?

Reply:
The FileAPI definition of Unix Epoch this links to kind of hints at that already... Also since this is only exposed via FileAPI currently, it is probably enough to rely on whatever text that spec has (and update it their if needed).